### PR TITLE
tricore

### PR DESCRIPTION
**Pull request by woruyu**

### CODE REVIEW

**Comment by dkm** [comment]
Could you tell us a bit more what this is about? There are some changes that must be your local setup...

File: etc/config/c++.defaults.properties [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Default settings for C++
-compilers=&gcc:&clang
+compilers=&gcc:&clang:&tasking
+
+group.tasking.compilers=Smartcode_v10
+group.tasking.compilerType=tasking
+compiler.Smartcode_v10.exe=C:\Users\QXZ3F7O\Desktop\Smartcode_v10.1r1\ctc\bin\hldumptc.exe
```
Comment (dkm) [inline etc/config/c++.defaults.properties:6]: this is local for example

File: etc/config/c.defaults.properties [hunk 2]
```diff
@@ -30,14 +35,16 @@ compiler.cg7.exe=/usr/bin/gcc-7
...
 compiler.cg9.name=gcc 9.x
 compiler.cg10.exe=/usr/bin/gcc-10
 compiler.cg10.name=gcc 10.x
-compiler.cg11.exe=/usr/bin/gcc-11
+compiler.cg11.exe=/Users/QXZ3F7O/MinGW/bin/g++.exe
 compiler.cg11.name=gcc 11.x
-compiler.cgdefault.exe=/usr/bin/gcc
+compiler.cgdefault.exe=/Users/QXZ3F7O/MinGW/bin/mingw32-gcc.exe
 compiler.cgdefault.name=gcc default
+compiler.cgtricore.exe=/Users/QXZ3F7O/Desktop/tricore_494_mingw/bin/tricore-elf-gcc.exe
```
Comment (dkm) [inline etc/config/c.defaults.properties:46]: cross compiler are not defined here. There is a specific config section below with a bunch of gcc-cross

File: etc/config/c.defaults.properties [hunk 3]
```diff
@@ -30,14 +35,16 @@ compiler.cg7.exe=/usr/bin/gcc-7
 compiler.cg7.name=gcc 7.x
 compiler.cg8.exe=/usr/bin/gcc-8
 compiler.cg8.name=gcc 8.x
-compiler.cg9.exe=/usr/bin/gcc-9
+compiler.cg9.exe=/Users/QXZ3F7O/MinGW/bin/mingw32-gcc-9.2.0.exe
```
Comment (dkm) [inline etc/config/c.defaults.properties:38]: This is probably not meant to be part of the changeset

File: lib/parsers/asm-parser-beebasm.js [hunk 4]
```diff
@@ -5,7 +5,6 @@ import {AsmParser} from './asm-parser';
 export class AsmParserBeebAsm extends AsmParser {
     constructor(compilerProps) {
         super(compilerProps);
-
```
Comment (dkm) [inline lib/parsers/asm-parser-beebasm.js:8]: please avoid reformatting unrelated parts

File: lib/parsers/asm-parser-cc65.ts [hunk 5]
```diff
@@ -135,17 +135,14 @@ export class CC65AsmParser extends AsmParser {
...
                 if (filters.directives) {
                     const directiveMatch = asmtext.match(this.directiveRe);
                     if (directiveMatch) continue;
                 }
-
                 result.push({
                     text: '  ' + asmtext,
                     labels: this.extractLabels(asmtext, 3),
                 });
-
```
Comment (dkm) [inline lib/parsers/asm-parser-cc65.ts:148]: same

**Comment by dkm** [comment]
@woruyu , I hope I did not scare you away with my comments, that was not the goal. If you need help with your changes, feel free to reopen, and we'll guide you through the process of getting your changes in shape :)

**Comment by woruyu** [comment]
hello,
I'm sorry for uploading this pull request, I made a mistake, I originally wanted to upload it to my repo , this is just a plug-in related to the Tasking compiler. And&nbsp; I didn't understand Godbolt very well, I just used its logic frame.
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "compiler-explorer/compiler-explorer"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年2月7日(星期二) 下午5:07
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [compiler-explorer/compiler-explorer] tricore (PR #4701)





 
@woruyu , I hope I did not scare you away with my comments, that was not the goal. If you need help with your changes, feel free to reopen, and we'll guide you through the process of getting your changes in shape :)

**Pull request closed without merging**